src/libs/ajax.js

```diff
@@ -80,22 +80,23 @@ const checkRequesterPaysError = async response => {
   }
 }
 
+export const canUseWorkspaceProject =
+  async ({ canCompute, workspace: { namespace } }) => canCompute || _.some(
+    p => p.projectName === namespace && _.includes('Owner', p.roles),
+    await Ajax().Billing.listProjects()
+  )
```

Comment:
We try not to break before a `=>` in a function declaration. How's this look?
```suggestion
export const canUseWorkspaceProject = async ({ canCompute, workspace: { namespace } }) => {
  return canCompute || _.some(
    ({ projectName, roles }) => projectName === namespace && _.includes('Owner', roles),
    await Ajax().Billing.listProjects()
  )
}
```